pkg/config/config.go

```diff
@@ -1595,6 +1595,19 @@ func (r *RuntimeHandler) RuntimeSupportsIDMap() bool {
 	return true
 }
 
+// RuntimeSupportsMountFlag returns whether this runtime supports the specified mount option.
+func (r *RuntimeHandler) RuntimeSupportsMountFlag(flag string) bool {
+	if r.features.MountOptions == nil {
+		return false
+	}
+	for _, m := range r.features.MountOptions {
```

Comment:
You can use the "slices" package and remove the loop. It seems crio is using go 1.21 already, that is the first release to include `slices.Contains()`

Reply:
good hint, fixed now